src/common/Icons.js

```diff
@@ -86,6 +86,9 @@ export const IconDiagnostics: SpecificIconType = makeIcon(Feather, 'activity');
 export const IconNotifications: SpecificIconType = makeIcon(Feather, 'bell');
 export const IconLanguage: SpecificIconType = makeIcon(Feather, 'globe');
 export const IconSettings: SpecificIconType = makeIcon(Feather, 'settings');
+// collapsible stream icons
+export const IconUnCollapse: SpecificIconType = makeIcon(FontAwesome, 'caret-up');
+export const IconCollapse: SpecificIconType = makeIcon(FontAwesome, 'caret-down');
```

Comment:
These names are potentially pretty confusing because it's hard to be sure if "icon collapse" means
- the icon *to collapse*, shown when the thing is uncollapsed
- the icon *when collapsed*, which you use to uncollapse the thing

Instead `IconCaretUp` and `IconCaretDown` would be good.

Reply:
Separately: though these are called "caret", the actual icons seen in your screencap aren't really carets but triangles.

We can get actual carets by using `MaterialIcon` here instead of `Feather`. For a model design, see "Expand and collapse" here:
https://material.io/components/lists

Reply:
Here is a screen shot with those icons from Material:

<img width="327" alt="Screen Shot 2022-09-02 at 10 40 42 AM" src="https://user-images.githubusercontent.com/50177345/188208682-4469b200-b50b-466c-80c5-2aa752acc7ce.png">